### ISSUE TITLE
julia.executablePath error with the last extension update on Ubuntu 22.04

### ISSUE THREAD

**Issue by RayleighLord** [julia-vscode/julia-vscode#3446]
I have julia installed via jill. Up until now, vscode was able to find the Julia version installed on my PC without any problem. However, as of the last update I am getting the error message:

```Could not start the Julia language server. Make sure the `julia.executablePath` setting points to the Julia binary.```

whenever I open any Julia project. Any ideas on what caused this process to not be automatic anymore?

**Comment by pfitzseb** [comment]
Can you paste the content of Outputs > Julia here? It should show something like
[Peek 2023-10-25 11-33.webm](https://github.com/JuliaComputing/JuliaHub/assets/6735977/33b2face-eccc-4ee1-876d-4c2ea683745d)

**Comment by RayleighLord** [comment]
```
Trying to locate Julia binary...
Juliaup not found, locating Julia by other means.
No Julia executable was identified.
The current PATH environment variable is "/home/javier/.local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin".
```

It seems to look first for Juliaup. As a side question, should I use Juliaup on Linux? The last time I checked it was only advised to use on Windows.

**Comment by pfitzseb** [comment]
Ok, so what do you have `juliia.executablePath` set to? If nothing, how is `julia` resolved in a terminal? Are you defining an alias in `.bashrc`? If so, start VS Code from bash and that should fix things.

**Comment by RayleighLord** [comment]
`julia` in a terminal opens a Julia session directly. jill by defaulf puts your executables in the folder `~/.local/bin/`, so if in `julia.executablePath` I set that path everything works perfectly. But I wanted to rise the concern that this process worked automatically up until today, in which I had to manually set the executable path.

Although if the recommended way to install julia now is via Juliaup, and this avoids the problem, I guess it is fine. But for users with jill now it seems to require manually setting the executable path.
